public/style/base/_reset.scss

```diff
@@ -0,0 +1,78 @@
+html {
+  // Prevent adjustments of font size after orientation changes in IE and iOS. */
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+a,
```

Comment:
A yeah, I didn;t want to mess too much with what you had and this was under the "reset" comment block